drake/examples/Quadrotor/runDynamics.cpp

```diff
@@ -1,25 +1,57 @@
 
 #include <iostream>
-#include "Quadrotor.h"
-#include "drake/systems/Simulation.h"
-#include "drake/systems/plants/BotVisualizer.h"
+#include "BotVisualizer.h"
```

Comment:
Definitely.  I thought we had updated all of them.  Perhaps the carsim came in parallel to twan's big update.  Thanks!